test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/MSBuildStaticGraphRestoreTests.cs

```diff
@@ -221,8 +221,11 @@ public void GetPackagesPath_WhenPathOrOverrideSpecified_PathIsCorrect(string pac
             {
                 var project = new MockMSBuildProject(testDirectory, new Dictionary<string, string>
                 {
-                    ["RestorePackagesPathOverride"] = packagesPathOverride,
                     ["RestorePackagesPath"] = packagesPath
+                },
+                new Dictionary<string, string>
```

Comment:
use named parameters. I have no idea what the difference between the two dictionaries are.